src/table/resizer/styles.scss

```diff
@@ -50,7 +50,8 @@ $active-separator-width: 2px;
       }
     }
   }
-  &.has-focus {
+  // stylelint-disable-next-line selector-combinator-disallowed-list
+  [data-awsui-focus-visible='true'] &.has-focus {
```

Comment:
Should this check for `body[data-awsui-focus-visible='true']` ?

Reply:
Correct. Fixed in the next revision